scripts/quantit_generate_driver_file.py

```diff
@@ -47,22 +47,22 @@ def make_location_dict(self, io_filtered):
         return location_dict
 
     def make_file(self, location_dict):
-        """Writes the formated well location info into a driver 
+        """Writes the formated well location info into a driver
         file sorted by row and col."""
-        keylist = location_dict.keys()
+        keylist = list(location_dict.keys())
         keylist.sort()
         f = open(self.drivf, 'a')
-        print >> f , 'Row,Column,*Target Name,*Sample Name'
+        print('Row,Column,*Target Name,*Sample Name', file=f)
         for key in keylist:
-            print >> f ,location_dict[key]
+            print(location_dict[key], file=f)
         f.close()
 
 def main(lims, pid, drivf ,epp_logger):
     process = Process(lims,id = pid)
     QiT = QuantitDriverFile(process, drivf)
     io = process.input_output_maps
-    io_filtered = filter(lambda (x,y): y['output-generation-type']=='PerInput', io)
-    io_filtered = filter(lambda (x,y): y['output-type']=='ResultFile', io_filtered)
+    io_filtered = [x_y for x_y in io if x_y[1]['output-generation-type']=='PerInput']
```

Comment:
Is there a reason the variable is named `x_y` and `x_y1` instead of just `x` or `y`?

Reply:
No, this is the suggested change by `2to3`

Reply:
👍 Ok!